### ISSUE TITLE
Unable to drag/reposition nodes in Brave (Chromium)

### ISSUE THREAD

**Issue by papakpmartin** [node-red/node-red-dashboard#827]
### Current Behavior

In Node-RED v3.1.3 using node-red-dashboard 3.6.2, it’s apparently not possible to drag/reposition nodes around using the latest Brave Browser (on Mac; v1.61.114 Chromium: 210.0.6099.199). Brave's "shields" are down. I tried clearing cache. No interesting errors in console when trying to select/drag.

Doing the same thing from the same server using Safari works as expected.

https://github.com/node-red/node-red/assets/3298248/fe19d20f-0ace-44ec-a636-4882e89046b7

https://github.com/node-red/node-red/assets/3298248/acb07491-9387-4a50-94c3-9e1ad7ff5e06

### Expected Behavior

Dragging nodes should work as seen in the Safari video

### Example flow

```
no minimal example available right now
```

### Environment

- Node-RED version: v3.3.1
- Node.js version: v18.19.0
- npm version: 10.2.3
- Platform/OS: Browsers running on Mac; Node-RED running on Raspberry
- Browser: Brave and Safari

**Comment by joepavitt** [comment]
My fault @knolleary - when explained in Slack I read this as not being able to drag the nodes in the Node-RED Editor, so asked to raise here.

**Comment by dceejay** [comment]
Yeah - the gridstack library we use was evolved really quickly after we adopted it, and they have broken the API multiple times since. (now v10.0 vs our v0.6). Last time I tried to update it, it broke a whole bunch of other stuff, and needed other support, so I have just pinned it back. The reality is that I am unlikely to ever fix this as we are slowly moving over to Dashboard2, but if anyone wants to have a go please jump in and do so, as it would be much appreciated.

**Comment by papakpmartin** [comment]
Is Dashboard2 available for use?

**Comment by joepavitt** [comment]
It most certainly is: https://dashboard.flowfuse.com/

Not 100% feature parity yet, but most of it (plus some new stuff) is in there

**Comment by papakpmartin** [comment]
Awesome! I really like that you're using Vue3... good stuff.

FWIW, I'm cool with closing this. :)
